starfish/compat.py

```diff
@@ -0,0 +1,100 @@
+import skimage
+from packaging import version
+
+if version.parse(skimage.__version__) > version.parse("0.14.2"):
```

Comment:
Yuck.  Can we just require >= 0.14.2?

Reply:
No, unfortunately we just missed an `skimage` release, and this is the most recent version.

Reply:
oh this is the code you added to skimage.